### DISCUSSION TITLE
Following the documentation isn't working

### DISCUSSION THREAD

**Discussion by ronaldohoch** [GrapesJS/grapesjs#3568] [Q&A] [answered]
Hello guys!

Following the documentation, at this point, it works.:
![image](https://user-images.githubusercontent.com/2287371/123434411-d206a080-d5a2-11eb-854b-e12c4060b653.png)

But, in the same point at localhost, it throws the error, using even the js file came from dist folder, a new build or the cdn js.:
![image](https://user-images.githubusercontent.com/2287371/123434823-44778080-d5a3-11eb-9b63-c478b1cd1663.png)

I don't know what i'm doing wrog :/

**Comment by AStoker** [comment, marked as answer]
Try to move the init for GrapesJS into a script tag at the bottom of the `<body>`. My guess is that since it's being called in the head, perhaps DOM elements aren't ready yet?

**Reply by ronaldohoch** [reply]
Omg! 🤦‍♂️
![image](https://user-images.githubusercontent.com/2287371/123441011-c5d21180-d5a9-11eb-97df-8471e26df712.png)

Thank you, i'm embarrassed now!
